src/RageTimer.cpp

```diff
@@ -88,6 +95,15 @@ float RageTimer::GetDeltaTime()
 	return diff;
 }
 
+/* This function exists because we don't _really_ need to calculate the
+time delta when the "Approximate sound time" error is being logged. It's
+faster to reset the timer and keep going. */
+void RageTimer::ResetTimer()
+{
+	const RageTimer Now;
```

Comment:
What the. All creating a new timer does is set `m_secs` and `m_us` to 0 and then calls `Touch()`.
https://github.com/itgmania/itgmania/blob/release/src/RageTimer.h#L11.

Since `Touch()` already overwrites `m_secs` and `m_us`, this function in general seems unnecessary and you can just call `Touch()` directly right?

Creating a new variable and reassigning `*this` is very odd.

Reply:
Good point, I didn't notice we could use `Touch()` for this.

To be honest, I don't think resetting the timer like this is necessary, but it maintains the behavior of mainline without also doing math which the value returned is never used -> https://github.com/itgmania/itgmania/blob/04647648765b082414993e6f2ce420b2d950840b/src/RageSoundPosMap.cpp#L184C3-L184C23

I would be happy to remove this ResetTimer() function and use `Touch()` instead.